### DISCUSSION TITLE
Why do I get this error message after I upgrade?

### DISCUSSION THREAD

**Discussion by soarcx** [oobabooga/text-generation-webui#5815] [Q&A]
I installed the documentation on Windows11. It was normal before I upgraded. After upgrading today, a message appeared. I am a novice. I hope someone can help me. Thank you.

─ Traceback (most recent call last) ──────────────────────────────────────────╮
│ D:\text-generation-webui-main\text-generation-webui-main\server.py:21 in <module>                                    │
│                                                                                                                      │
│    20 with RequestBlocker():                                                                                         │
│ ❱  21     from modules import gradio_hijack                                                                          │
│    22     import gradio as gr                                                                                        │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\modules\gradio_hijack.py:9 in <module>                      │
│                                                                                                                      │
│    8                                                                                                                 │
│ ❱  9 import gradio as gr                                                                                             │
│   10                                                                                                                 │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\__init__.py:3  │
│ in <module>                                                                                                          │
│                                                                                                                      │
│     2                                                                                                                │
│ ❱   3 import gradio._simple_templates                                                                                │
│     4 import gradio.image_utils                                                                                      │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\_simple_templa │
│ tes\__init__.py:1 in <module>                                                                                        │
│                                                                                                                      │
│ ❱ 1 from .simpledropdown import SimpleDropdown                                                                       │
│   2 from .simpleimage import SimpleImage                                                                             │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\_simple_templa │
│ tes\simpledropdown.py:6 in <module>                                                                                  │
│                                                                                                                      │
│     5                                                                                                                │
│ ❱   6 from gradio.components.base import FormComponent                                                               │
│     7 from gradio.events import Events                                                                               │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\components\__i │
│ nit__.py:40 in <module>                                                                                              │
│                                                                                                                      │
│    39 from gradio.components.model3d import Model3D                                                                  │
│ ❱  40 from gradio.components.multimodal_textbox import MultimodalTextbox                                             │
│    41 from gradio.components.number import Number                                                                    │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\components\mul │
│ timodal_textbox.py:34 in <module>                                                                                    │
│                                                                                                                      │
│    33 @document()                                                                                                    │
│ ❱  34 class MultimodalTextbox(FormComponent):                                                                        │
│    35     """                                                                                                        │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\component_meta │
│ .py:198 in __new__                                                                                                   │
│                                                                                                                      │
│   197         component_class = super().__new__(cls, name, bases, attrs)                                             │
│ ❱ 198         create_or_modify_pyi(component_class, name, events)                                                    │
│   199         return component_class                                                                                 │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\site-packages\gradio\component_meta │
│ .py:92 in create_or_modify_pyi                                                                                       │
│                                                                                                                      │
│    91                                                                                                                │
│ ❱  92     source_code = source_file.read_text()                                                                      │
│    93                                                                                                                │
│                                                                                                                      │
│ D:\text-generation-webui-main\text-generation-webui-main\installer_files\env\Lib\pathlib.py:1059 in read_text        │
│                                                                                                                      │
│   1058         with self.open(mode='r', encoding=encoding, errors=errors) as f:                                      │
│ ❱ 1059             return f.read()                                                                                   │
│   1060                                                                                                               │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
UnicodeDecodeError: 'gbk' codec can't decode byte 0xb2 in position 2074: illegal multibyte sequence

**Comment by devPhases** [comment]
I just tried updating today, for like the fourth time over the past three weeks. It hasn't worked for me in three weeks :(

I guess we get left behind so long as it works for others.

**Comment by FartyPants** [comment]
Don't upgrade, rename your old folder to something like _old and install new
